src/compass/utils/lut.py

```diff
@@ -11,6 +11,13 @@ def compute_geocoding_correction_luts(burst, rg_step=200, az_step=0.25):
         LUT spacing (in pixels) along x/slant range in meters
     ystep: int
         LUT spacing (in pixels) along x/azimuth in seconds
+    path_dem: str
+        Path to the DEM required for azimuth FM rate mismatch.
+        If `None`, the calculation of the FM mismatch will be skipped.
+    path_scratch: str
```

Comment:
I would suggest using `dem_path` and `scratch_path` in style, to be more consistent with existing ones: https://github.com/opera-adt/s1-reader/blob/ae88a1d56df19b8e1cd5f8be9bc73e195e256c31/src/s1reader/s1_reader.py#L472-L473